vendor/bat-native-ledger/src/logging.h

```diff
@@ -12,11 +12,6 @@
 #define BVLOG(client, severity) \
   client->LogVerbose(__FILE__, __LINE__, severity)->stream()
 
-#endif  // BAT_REWARDS_LOGGING_H_
+#include "base/logging.h"
```

Comment:
what are changes in this file?

Reply:
This change is to allow `url/gurl` to be included and not clash with `DCHECK`

Reply:
please ensure that both `//base` and `//url` are added as deps in BUILD.gn

Reply:
Done.